examples/proc_FIR.py

```diff
@@ -67,51 +75,72 @@
                 fl.next("after auto-clock correction")
                 fl.image(s)
             # }}}
-        np.squeeze(s)
-        s, _ = prscr.rough_table_of_integrals(s, fl=fl)
+        s.squeeze()
```

Comment:
squeeze affects the dimensions, so we need to use the pyspecdata method.  I'm actually surprised this didn't give an error, but hopefullly you can see how squeeze is different from e.g. sqrt or log, etc.

Reply:
what do you mean by this? should I be returning the list of dropped dimensions?